net/croc/Makefile

```
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2023 Jonas Jelonek
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=croc
+PKG_VERSION:=9.6.5
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/schollz/croc.git
+PKG_MIRROR_HASH:=324ab1ef98b584904391e55b4f37a1a24fb089303c49a27eeeb4881e7d2d2e82
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
```

```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://codeload.github.com/schollz/croc/tar.gz/v$(PKG_VERSION)?
PKG_HASH:=2d3ba7bae3c49e3870e2f8523c6be00e92fe6e46828269a8cea34d4034102cad
```

it's recommended to use codeload instead.

---

done